src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -1071,9 +1073,10 @@ PeerImp::onMessage (std::shared_ptr <protocol::TMTransaction> const& m)
             std::weak_ptr<PeerImp> weak = shared_from_this();
             app_.getJobQueue ().addJob (
                 jtTRANSACTION, "recvTransaction->checkTransaction",
-                [weak, flags, stx] (Job&) {
+                [weak, flags, checkSignature, stx] (Job&) {
```

Comment:
@HowardHinnant `RIPPLED_USE_CPP_14` has no effect on Visual Studio builds (other than to change the contents of some beast headers)